glib/src/value.rs

```diff
@@ -930,6 +936,33 @@ impl SetValueOptional for str {
     }
 }
 
+impl<'a> FromValue<'a> for Value {
+    unsafe fn from_value(value: &'a Value) -> Self {
+        from_glib_none(
+            gobject_ffi::g_value_get_boxed(value.to_glib_none().0) as *mut gobject_ffi::GValue
```

Comment:
That works, is correct and also has the lowest number of copies :)

For more correctness:

```suggestion
            gobject_ffi::g_value_get_boxed(value.to_glib_none().0) as *const gobject_ffi::GValue
```


You could also add a

```rust
impl<'a> FromValue<'a> for &'a Value {
    unsafe fn from_value(value: &'a Value) -> Self {
        &*(
            gobject_ffi::g_value_get_boxed(value.to_glib_none().0) as *const Value
        )
    }
}
```

And the corresponding `FromValueOptional` impl